test/dataframe.jl

```diff
@@ -377,6 +398,36 @@ module TestDataFrame
         a = unstack(df, :id, :variable, :value)
         b = unstack(df, :variable, :value)
         @test a ≅ b ≅ DataFrame(id = [1, 2], a = [3, missing], b = [missing, 4])
+
+        df = DataFrame(variable=["x", "x"], value=[missing, missing], id=[1,1])
+        @test_warn "Duplicate entries in unstack at row 2." unstack(df, :variable, :value)
+        @test_warn "Duplicate entries in unstack at row 2." unstack(df)
+    end
+
+    @testset "missing values in colkey" begin
+        df = DataFrame(id=[1, 1, 1, missing, missing, missing, 2, 2, 2],
```

Comment:
Spaces around `=`.

Reply:
I thought the style without spaces is recommended by Julia Manual. Also you do not have spaces in this very file (see e.g. at its beginning - most of calls to `DataFrame` do not use spaces here).
I can fix it - but let us decide - do we want spaces or no spaces everywhere?
(adding spaces will change much more lines)

Reply:
OK, it's already inconsistent, so let's leave it that way.